apps/frontend/src/pages/MainPage.tsx

```diff
@@ -95,6 +102,7 @@ export const MainPage = () => {
     onCloseWorkoutEditorModal,
     onOpenWelcomeMessageModal,
     onCloseWelcomeMessageModal,
+    optionsModal.isOpen,
```

Comment:
Is this needed? Seems like all the other dependencies are `onOpen` and `onClose`?
```suggestion
    optionsModal.onOpen,
    optionsModal.onClose,
```

Reply:
No, good catch. Fixed now.